core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/autocompletecoral3/AutocompleteOption.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.api.annotations.widgets.autocompletecoral3;
+
+import com.exadel.aem.toolkit.api.annotations.meta.PropertyRendering;
+
+/**
+ * Used to define an option within the {@link Autocomplete#options()} set
+ * See documentation on <a href="https://developer.adobe.com/experience-manager/reference-materials/6-5/coral-ui/coralui3/Coral.Autocomplete.html">
+ * Autocomplete</a> component
+ */
+public @interface AutocompleteOption {
```

Comment:
We need to cover all of these attributes https://developer.adobe.com/experience-manager/reference-materials/6-5/coral-ui/coralui3/Coral.Autocomplete.html#Coral.Autocomplete.Item:api. They do exist and make a difference. E.g., the "selected" attr will surely be demanded
![image](https://user-images.githubusercontent.com/48795460/187700607-c677fe1f-e6b6-4bcd-b497-96db50fc880c.png)

Reply:
BTW, maybe `AutocompleteItem` and the collection named `items`? To not walk to far away from the API

Reply:
done